docs/concepts/consistency.mdx

```diff
@@ -9,7 +9,7 @@ TopK supports three different consistency levels, allowing you to choose the rig
 TopK follows a write-ahead log (WAL) and compaction-based architecture:
 
 1. **Writer** appends incoming data to the **WAL**.
-2. **Compactor** processes the WAL and produces optimized, indexed files.
+2. **Compactor** processes the WAL and produces optimized, indexed files, also known as *compacted files*.
```

Comment:
a mini diagram would be nice to show the data flow. like here https://www.topk.io/resources/engineering-journal